weblab_cg/_generator_chaste_cg.py

```diff
@@ -74,30 +74,28 @@ class ChasteModel(object):
                    'units': str(self._model.units.summarise_units(self._membrane_capacitance.lhs))}]
                  )}}
 
-    def __init__(self, model, class_name, file_name, **kwargs):
+    def __init__(self, model, cls_name, file_name, _header_ext='.hpp', **kwargs):
```

Comment:
Having a parameter name with leading underscore is odd.

Also, why rename class_name? Especially when the member variable isn't renamed?

Reply:
I wanted to avoid classh with class_name in the kwargs but have changed it to use that class_name instead now.